{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/auth/api.py

```diff
@@ -64,3 +74,42 @@ def password_reset_confirm(self, request):
         user.set_password(serializer.validated_data['new_password'])
         user.save()
         return response.NoContent()
+
+
+class CurrentUserViewSet(viewsets.GenericViewSet):
+    serializer_class = serializers.UserSerializer
+    queryset = User.objects.filter(is_active=True)
+    permission_classes = (IsAuthenticated, )
+
+    def get_object(self):
+        return self.request.user
+
+    @list_route(['GET', ])
+    def get_current_user_profile(self, request):
+        """
+        Get logged in user profile
+        """
+        ctx = {'request': request}
+        serializer = self.get_serializer(self.get_object(), context=ctx)
+        return response.Ok(serializer.data)
+
+    @list_route(['PUT', ])
+    def udpate_current_user_profile(self, request, *args, **kwargs):
+        """
+        Update logged in user profile
+        """
+        partial = kwargs.pop('partial', False)
+        instance = self.get_object()
+        ctx = {'request': request}
```

Comment:
You don't have to pass `request` context as its passed by `get_serializer` method by default.